test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13932,11 +13931,12 @@ def create_script(self, body=None, asuser=None, hostname=None):
                     body[i] = " ".join(line_arr)
             body = '\n'.join(body)
 
+        # If the user has a userhost, the job will run from there
+        # so the script should be made there
```

Comment:
This sounds like a pretty generic logic, so can we remove "if self.platform == 'shasta'" from the check below?